Kitodo/src/main/java/de/sub/goobi/forms/ProzesskopieForm.java

```diff
@@ -817,9 +818,32 @@ public String createNewProcess() throws ReadException, IOException, PreferencesE
 
         startTaskScriptThreads();
 
+        if (Config.getBooleanParameter("createDummyImagesAtProcessCreation")) {
+            createDummyImages();
+        }
+
         return processListPath;
     }
 
+    private void createDummyImages() throws IOException {
+        URI processDirectory = serviceManager.getFileService().getProcessBaseUriForExistingProcess(this.prozessKopie);
+        String imageDirectoryName = this.prozessKopie.getTitle() + "_" + Config.getParameter("MetsEditorDefaultSuffix");
```

Comment:
Maybe you should look into "logic of detecting directory" which is really used for meta data editor. Not in any case used directory is named with process title.

Reply:
There is a method in the file service for it.
getImagesDirectory(process)